src/metrics/include/metrics/Period.hpp

```diff
@@ -65,6 +65,19 @@ class Period final {
         return os << period._duration.count();
     }
 
+    std::chrono::seconds::rep getSecondsCount() const {
+        return std::chrono::duration_cast<std::chrono::seconds>(_duration).count();
+    }
+
+    /**
+     * Gets the nanosecond component that wouldn't be counted in seconds.
+     */
+    std::chrono::nanoseconds::rep getNanosecondsCount() const {
+        return std::chrono::duration_cast<std::chrono::nanoseconds>(_duration).count() -
+            std::chrono::duration_cast<std::chrono::nanoseconds>(std::chrono::duration_cast<std::chrono::seconds>(_duration)).count();
```

Comment:
yikes - your call consider `using namespace std::chrono` at the top of this function 😄